src/main/java/org/opentripplanner/apis/gtfs/datafetchers/LegImpl.java

```diff
@@ -189,10 +190,12 @@ public DataFetcher<Boolean> realTime() {
     return environment -> getSource(environment).getRealTime();
   }
 
-  // TODO
   @Override
   public DataFetcher<String> realtimeState() {
```

Comment:
I just realised that this doesn't use a proper mapper, which I think is a very useful thing.

Since I already approved, I wrote the code myself: https://github.com/leonardehrenfried/OpenTripPlanner/commit/a763bc5e3b05c361ad60f556201a5e6598b4478e

If you want you can cherry pick it into this PR.

Reply:
Ah, too late. I will open another one myself.